README.md

```diff
@@ -122,23 +122,42 @@ my_email: "scriptworker@example.com"
   "provisionerId": "test-dummy-provisioner",
   "workerType": "dummy-worker-jlund",
   "schedulerId": "-",
-  "taskGroupId": "S-lth0jTThKBjmpt386kUA",
+  "taskGroupId": "YkE_OQjoTj2G92P4A4Cy1A",
   "dependencies": [
-    "YVq4WkdlTmSz4on_FwuGIw"
+    "c6DPRNsTRa6uVYVp5qCD9w",
+    "M4hnMSqcS4itJWC-IwsmXg"
   ],
   "requires": "all-completed",
   "routes": [],
   "priority": "normal",
   "retries": 5,
-  "created": "2016-09-28T02:38:47.963Z",
-  "deadline": "2016-09-28T03:38:47.963Z",
+  "created": "2017-01-08T17:06:07.800Z",
+  "deadline": "2017-01-08T18:06:07.800Z",
   "expires": "2017-08-31T23:20:18.165Z",
   "scopes": [],
   "payload": {
-    "upload_date": 1472747174,
-    "update_manifest": false, # if true, create balrog prop artifact. used by signing beetmover task
-    "taskid_of_manifest": "YVq4WkdlTmSz4on_FwuGIw",  # used to determine template_key, version, etc
-    "taskid_to_beetmove": "YVq4WkdlTmSz4on_FwuGIw"
+    "maxRunTime": 600,
+    "upload_date": 1483465253,
+    "update_manifest": true,
+    "upstreamArtifacts": [
+      {
+        "paths": [
+          "public/build/balrog_props.json"
+        ],
+        "taskId": "c6DPRNsTRa6uVYVp5qCD9w",
+        "taskType": "build",
+        "locale": "en-US"
```

Comment:
Wondering if we will ever have different locale lists between the balrog_props and the other artifacts.  If so, having locales specified per `upstreamArtifact` dict makes sense.  If not, specifying them outside might make sense.